media-gfx/blender/blender-2.78c.recipe

```diff
@@ -105,10 +107,24 @@ BUILD()
 INSTALL()
 {
 	cd build_haiku
-	make install
+	make install $jobArgs
+
+	strip $INSTALL_PATH/blender
+
+	local MAJOR="`echo "$portVersion" | cut -d. -f1`"
+	local MIDDLE="`echo "$portVersion" | cut -d. -f2 | sed 's/[a-z]*//g'`"
+	local MINOR="`echo 0`"
+	sed \
+		-e "s|@MAJOR@|$MAJOR|" \
+		-e "s|@MIDDLE@|$MIDDLE|" \
+		-e "s|@MINOR@|$MINOR|" \
+		$portDir/additional-files/blender.rdef.in > blender.rdef
+
+	addResourcesToBinaries blender.rdef \
+		$INSTALL_PATH/blender
 
```

Comment:
How about renaming blender to Blender? Also, do other files in Blender dir needed?